sn_node_manager/src/bin/cli/main.rs

```diff
@@ -426,34 +460,108 @@ async fn main() -> Result<()> {
+                }
 
-            let local_node_reg_path = &get_local_node_registry_path()?;
-            let mut local_node_registry = NodeRegistry::load(local_node_reg_path)?;
-            if !local_node_registry.nodes.is_empty() {
-                return Err(eyre!("A local network is already running")
-                    .suggestion("Use the kill command to destroy the network then try again"));
+                if verbosity != VerbosityLevel::Minimal {
+                    println!("=================================================");
+                    println!("              Add Faucet Service                 ");
+                    println!("=================================================");
+                }
+
+                let service_user = "safe";
+                let service_manager = NodeServiceManager {};
+                service_manager.create_service_user(service_user)?;
+
+                let service_log_dir_path =
+                    get_service_log_dir_path(ReleaseType::Faucet, log_dir_path, service_user)?;
+
+                let mut node_registry = NodeRegistry::load(&get_node_registry_path()?)?;
+                let release_repo = <dyn SafeReleaseRepositoryInterface>::default_config();
+
+                let (faucet_download_path, version) = download_and_extract_release(
+                    ReleaseType::Faucet,
+                    url.clone(),
+                    version,
+                    &*release_repo,
+                )
+                .await?;
+
+                add_faucet(
+                    AddFaucetServiceOptions {
+                        bootstrap_peers: get_peers_from_args(peers).await?,
+                        env_variables,
+                        faucet_download_bin_path: faucet_download_path,
+                        faucet_install_bin_path: PathBuf::from("/usr/local/bin/faucet"),
+                        local: false,
+                        service_data_dir_path: get_faucet_data_dir(),
+                        service_log_dir_path,
+                        url,
+                        user: service_user.to_string(),
+                        version,
+                    },
+                    &mut node_registry,
+                    &service_manager,
+                    verbosity,
+                )
+                .await?;
+
+                Ok(())
             }
+            FaucetSubCmd::Start {} => {
+                if !is_running_as_root() {
+                    return Err(eyre!("The start command must run as the root user"));
+                }
 
-            let release_repo = <dyn SafeReleaseRepositoryInterface>::default_config();
-            let faucet_path =
-                get_bin_path(build, path, ReleaseType::Faucet, version, &*release_repo).await?;
+                let mut node_registry = NodeRegistry::load(&get_node_registry_path()?)?;
+                if let Some(faucet) = node_registry.faucet.clone() {
```

Comment:
Just a tiny nit, but marking the var as `mut` can help us remove the following line, `let mut faucet = faucet`.
```suggestion
                if let Some(mut faucet) = node_registry.faucet.clone() {
```